src/pages/workspace/accounting/intacct/import/SageIntacctImportPage.tsx

```diff
@@ -0,0 +1,140 @@
+import React, {useMemo} from 'react';
+import ConnectionLayout from '@components/ConnectionLayout';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {clearSageIntacctMappingsErrorField, clearSageIntacctTaxErrorField, updateSageIntacctBillable, updateSageIntacctSyncTaxConfiguration} from '@libs/actions/connections/SageIntacct';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import {getCurrentXeroOrganizationName} from '@libs/PolicyUtils';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import ROUTES from '@src/ROUTES';
+import type {SageIntacctMappingValue} from '@src/types/onyx/Policy';
+
+function getDisplayTypeTranslationKey(displayType?: SageIntacctMappingValue): TranslationPaths | undefined {
+    switch (displayType) {
+        case CONST.SAGE_INTACCT_CONFIG.MAPPING_VALUE.DEFAULT: {
+            return 'workspace.intacct.employeeDefault';
+        }
+        case CONST.SAGE_INTACCT_CONFIG.MAPPING_VALUE.TAG: {
+            return 'workspace.accounting.importTypes.TAG';
+        }
+        case CONST.SAGE_INTACCT_CONFIG.MAPPING_VALUE.REPORT_FIELD: {
+            return 'workspace.accounting.importTypes.REPORT_FIELD';
+        }
+        case CONST.SAGE_INTACCT_CONFIG.MAPPING_VALUE.NONE: {
+            return undefined;
+        }
+        default: {
+            return undefined;
+        }
+    }
+}
+
+function SageIntacctImportPage({policy}: WithPolicyProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const policyID: string = policy?.id ?? '-1';
+    const sageIntacctConfig = policy?.connections?.intacct?.config;
+    const config = policy?.connections?.intacct?.config;
+
+    const currentXeroOrganizationName = useMemo(() => getCurrentXeroOrganizationName(policy ?? undefined), [policy]);
+
+    const mapingItems = useMemo(
+        () =>
+            Object.values(CONST.SAGE_INTACCT_CONFIG.MAPPINGS).map((mapping) => {
+                const menuItemTitleKey = getDisplayTypeTranslationKey(sageIntacctConfig?.mappings?.[mapping]);
+                return {
+                    description: translate('workspace.common.mappingTitle', mapping, true),
+                    action: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_TOGGLE_MAPPINGS.getRoute(policyID, mapping)),
+                    title: menuItemTitleKey ? translate(menuItemTitleKey) : undefined,
```

Comment:
<img width="383" alt="Screenshot 2024-07-02 at 12 39 30" src="https://github.com/Expensify/App/assets/88395093/47d3798e-ba75-4a63-8a16-c5abd1b466f1">
It would look like this, I confirmed this with design team a while ago, so it should be fine.

Reply:
I can't resolve here so please do 🙏